README.md

```diff
@@ -392,22 +392,41 @@ your-secretRootKey-value:
+|:-----------------|:---------|:----------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `enabled` | bool | Whether the binding rotation option is enabled.                                                                                                           |                                                                                                                                                         |
+| `rotationFrequency` | string | Indicates the interval between credential rotations. The actual frequency may be longer than the one you specified.  See details under **Notes** below. | "m", "h"                                                                                                                                                |                                                                                                                                                                |
+| `rotatedBindingTTL`   |  string | Specifies for how long to keep the old `ServiceBinding`. The actual TTL may be longer than the one you specified.  See details under **Notes** below.   | "m", "h"                                                                                                                                                |   
 
-**Note:**<br> It isn't possible to enable automatic credentials rotation to an already-rotated `ServiceBinding` (with the `services.cloud.sap.com/stale` label).
+### Example
+In the following example, the `ServiceBinding` resource is configured to rotate credentials every 25 days and to keep the old `ServiceBinding` for 2 days before deleting it.
+```yaml
+apiVersion: services.cloud.sap.com/v1
+kind: ServiceBinding
+metadata:
+  name: sample-binding
+spec:
+  serviceInstanceName: sample-instance
+  credentialsRotationPolicy:
+    enabled: true
+    rotatedBindingTTL: 48h
+    rotationFrequency: 600h
+ ```
+
+**Notes**
+* **The `credentialsRotationPolicy` is evaluated and executed during the [control loop](https://kubernetes.io/docs/concepts/architecture/controller/) which runs on every update or during
+  a full reconciliation process. This means that the actual rotation time may potentially be greater than the specified duration in 'rotationFrequency'.**
+* The original `ServiceBinding` and `Secret` will always contain the newest BindingID and credentials, old BindingID and credentials are backed up in a new `ServiceBinding` and `Secret` and will be deleted according to the configured TTL. The name of the backup `ServiceBinding` is '<original-binding-name>-guid'
```

Comment:
This is not a note, but description of the process. Which shall be elaborated above.
Need to explain what happens on binding rotation. 

the explanation in this bullet is good, but can be refined - mainly and the order

- A new service binding with the same name is created
- The old binding is renamed to original-binding-name-guid
- The new credentials are populated to the original secret
- The old credentials are kept in a temporary secret, which will be deleted ted according to the configured TTL.